governance/votes/Executive vote - March 3, 2023

```diff
@@ -0,0 +1,97 @@
+
+---
+
+## Proposal Details
+
+### Vault Parameter Changes
+
+As per this [post](https://forum.makerdao.com/t/emergency-proposal-risk-and-governance-parameter-changes-11-march-2023/20125) from the Risk Core Unit (RISK-001), the following emergency parameter changes will be made, if this executive proposal passes:
+
+#### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Changes
+
+- Reduce the UNIV2USDCETH-A `line` by 50 million DAI from 50 million DAI to **0 DAI**.
+- Reduce the UNIV2DAIUSDC-A `line` by 100 million DAI from 100 million DAI to **0 DAI**.
+- Reduce the GUNIV3DAIUSDC1-A `line` by 100 million DAI from 100 million DAI to **0 DAI**.
+- Reduce the GUNIV3DAIUSDC2-A `line` by 100 million DAI from 100 million DAI to **0 DAI**.
+
+### PSM Parameter Changes
+
+As per this [post](https://forum.makerdao.com/t/emergency-proposal-risk-and-governance-parameter-changes-11-march-2023/20125) from the Risk Core Unit (RISK-001), the following emergency parameter changes will be made, if this executive proposal passes:
+
+#### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change
+
+- Increase the PSM-USDP-A `line` by 550 million DAI to **1 billion DAI**.
+
+#### [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) Changes
+
+- Increase the PSM-USDC-A `tin` by 1% from 0% to **1%**.
+- Reduce the PSM-USDP-A `tin` by 0.2% from 0.2% to **0%**.
+
+#### [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) Change
+
+- Increase the PSM-USDP-A `tout` by 1% from 0% to **1%**.
+
+#### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Changes
+
+- Reduce the PSM-USDC-A `gap` by 700 million DAI from 950 million DAI to **250 million DAI**.
+- Reduce the PSM-GUSD-A `gap` by 40 million DAI from 50 million DAI to **10 million DAI**.
+- Increase the PSM-USDP-A `gap` by 200 million DAI from 50 million DAI to **250 million DAI**.
+
+### DAI Direct Deposit Module (D3M) [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar) Changes
+
+**Note that the proposed changes to the D3M `bar` parameters are not subject to the GSM Pause Delay due to the presence of a MOM contract. Therefore, these changes will be actioned when the executive proposal is scheduled**.
+
+As per this [post](https://forum.makerdao.com/t/emergency-proposal-risk-and-governance-parameter-changes-11-march-2023/20125) from the Risk Core Unit (RISK-001), the following emergency parameter changes will be made, if this executive proposal passes:
+
+- Reduce the DIRECT-AAVEV2-DAI `bar` by 2% from 2% to **0%**.
+- Reduce the DIRECT-COMPV2-DAI `bar` by 2% from 2% to **0%**.
+
```

Comment:
Not sure if it's Makerburn that's incorrect but it says the Aave D3M bar is already 0% and Compound D3M bar is 1.98%. Links: https://makerburn.com/#/collateral/direct_aavev2_dai/params and https://makerburn.com/#/collateral/direct_compv2_dai/params

Reply:
it's a makerburn issue